src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1982,6 +2001,43 @@ - (BOOL)accessibilityIsContentsAttributeSettable
 
 @end
 
+// these constants are duplicated in CAccessibility.java
+#define JAVA_AX_ROWS (1)
+#define JAVA_AX_COLS (2)
+
+@implementation TableAccessibility
+
+- (NSArray *)initializeAttributeNamesWithEnv:(JNIEnv *)env
+{
+    NSMutableArray *names = (NSMutableArray *)[super initializeAttributeNamesWithEnv:env];
+
+    [names addObject:NSAccessibilityRowCountAttribute];
+    [names addObject:NSAccessibilityColumnCountAttribute];
+    return names;
+}
+
+- (id)getTableInfo:(jint)info {
+    if (fAccessible == NULL) return 0;
+
+    JNIEnv* env = [ThreadUtilities getJNIEnv];
+    GET_TABLEINFO_METHOD_RETURN(nil);
+    jint count = (*env)->CallStaticIntMethod(env, jm_getTableInfo, fAccessible,
+                                             fComponent, info);
+    CHECK_EXCEPTION();
```

Comment:
I'm not sure you need this exception check, but it does no harm so it might be better to leave it as-is.

Reply:
The check is usually added when replacing the JNF calls (see other usages of CallStaticIntMethod). I also think it's better to have it even though the method should usually not throw any Exceptions. Thanks for the review!